<launch>
  <arg name="input_image" default="/spot/camera/back/image/compressed"/>
  <arg name="brightness_threshold" default="60" />
  <arg name="rest_time_per_hour" default="5" /> # min

  # person attention
  <arg name="panorama_image" default="/dual_fisheye_to_panorama/output/quater" />
  <arg name="human_detection_result" default="/spot_recognition/rects" />
  <arg name="human_width_threshold" default="0.45" /> # rate of human(bbox) height
  <arg name="human_height_threshold" default="0.4" /> # rate of image height
  <arg name="max_tracking_yaw" default="0.8" /> # radian

  <node name="spot_watchdog" pkg="jsk_spot_watch_dog" type="watch_dog.py"
        output="screen" respawn="false">
    <remap from="~input" to="$(arg input_image)" />
    <remap from="~panorama" to="$(arg panorama_image)" />
    <remap from="~human" to="$(arg human_detection_result)" />
    <param name="brightness_threshold" value="$(arg brightness_threshold)" />
    <param name="rest_time_per_hour" value="$(arg rest_time_per_hour)" />
    <param name="human_width_threshold" value="$(arg human_width_threshold)" />
    <param name="human_height_threshold" value="$(arg human_height_threshold)" />
    <param name="max_tracking_yaw" value="$(arg max_tracking_yaw)" />
  </node>
</launch>




